test/new-e2e/tests/installer/unix/package_agent_test.go

```diff
@@ -461,3 +463,69 @@ func (s *packageAgentSuite) installDebRPMAgent() {
 	}
 
 }
+
+func (s *packageAgentSuite) TestInstallWithDDOT() {
+	// Install datadog-agent (base infrastructure)
+	s.RunInstallScript("DD_REMOTE_UPDATES=true", envForceInstall("datadog-agent"))
+	defer s.Purge()
+	s.host.AssertPackageInstalledByInstaller("datadog-agent")
+	s.host.WaitForUnitActive(s.T(), agentUnit, traceUnit)
+
+	// Install ddot
+	s.host.Run(fmt.Sprintf("sudo datadog-installer install oci://installtesting.datad0g.com/ddot-package:pipeline-%s", os.Getenv("E2E_PIPELINE_ID")))
+	s.host.AssertPackageInstalledByInstaller("datadog-agent-ddot")
+
+	// Check if datadog.yaml exists, if not return an error
+	s.host.Run("sudo test -f /etc/datadog-agent/datadog.yaml || { echo 'Error: datadog.yaml does not exist'; exit 1; }")
+
+	// Substitute API & site into otel-config.yaml
+	apiKey := os.Getenv("DD_API_KEY")
+	if apiKey == "" {
+		apiKey = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+	}
+	site := os.Getenv("DD_SITE")
+	if site == "" {
+		site = "datadoghq.com"
+	}
+	s.host.Run(fmt.Sprintf("sudo sh -c \"sed -i -e 's/\\${env:DD_API_KEY}/%s/' -e 's/\\${env:DD_SITE}/%s/' /etc/datadog-agent/otel-config.yaml\"", apiKey, site))
+
+	s.host.WaitForUnitActive(s.T(), ddotUnit)
+
+	// Verify running
+	state := s.host.State()
+	s.assertUnits(state, false)
+	s.assertDdotUnits(state, false)
+
+	// Verify files exist
+	state.AssertFileExists("/etc/datadog-agent/otel-config.yaml", 0644, "dd-agent", "dd-agent")
+
+	state.AssertDirExists("/opt/datadog-packages/datadog-agent-ddot/stable", 0755, "dd-agent", "dd-agent")
+	state.AssertFileExists("/opt/datadog-packages/datadog-agent-ddot/stable/embedded/bin/otel-agent", 0755, "dd-agent", "dd-agent")
+
+	s.host.Run("sudo grep -q 'otelcollector:' /etc/datadog-agent/datadog.yaml")
+}
+
+// Verify ddot service running
+func (s *packageAgentSuite) assertDdotUnits(state host.State, oldUnits bool) {
```

Comment:
nit: Would prefer this function to be `assertDDOTUnits` (all same casing for DDOT)